plugins/woocommerce-admin/client/marketing/components/CollapsibleCard/CollapsibleCard.scss

```diff
@@ -0,0 +1,15 @@
+.woocommerce-collapsible-card {
+	.components-card-header {
+		font-size: 20px;
```

Comment:
In here and the other scss file. I'm wondering if we can use some variables instead of px hardcoded units  .

Reply:
Yeah I thought about that too. I looked into the WooCommerce code base, most of the CSS are using hardcoded units like this, so I just followed the existing way.

There are two other ways that doesn't directly use the hardcoded units:

- Using mixin: `@include font-size( 14 );`. Example: https://github.com/woocommerce/woocommerce/blob/fbb6bd1c53a7676b308b79a9051d475dd0e64c4c/plugins/woocommerce-admin/client/profile-wizard/steps/theme/style.scss#L35
- Using CSS variable like `font-size: var(--wp--preset--font-size--medium);`. Example: https://github.com/woocommerce/woocommerce/blob/fbb6bd1c53a7676b308b79a9051d475dd0e64c4c/plugins/woocommerce/client/legacy/css/woocommerce-blocktheme.scss#L161
    - Note that this approach is only used in codes in a folder named "legacy". I'm not sure if we should use this or avoid it.